### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by remko**

### CODE REVIEW

**Comment by mjnheather304** [comment]
Hey wrong bad guy I want Rich man who plays golf

File: docs/APIReference-KeyBindingUtil.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+id: api-reference-key-binding-util
+title: KeyBindingUtil
+layout: docs
+category: API Reference
+next: api-reference-modifier
+permalink: docs/api-reference-key-binding-util.html
+---
+
+The `KeyBingingUtil` module is a static set of utility functions for
```
Comment (hellendag) [inline docs/APIReference-KeyBindingUtil.md]: Spelling: `KeyBindingUtil`

File: docs/APIReference-KeyBindingUtil.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### isCtrlKeyCommand
+
+```
+isCtrlKeyCommand: function(
+  e: SyntheticKeyboardEvent
+): boolean
+```
+
+Check whether the ctrlKey modifier is *not* being used in conjunction with
+the altKey modifier. If they are combined, the result is an `altGraph`
```
Comment (hellendag) [inline docs/APIReference-KeyBindingUtil.md]: `ctrlKey`, `altKey`

**Comment by hellendag** [comment]
:100:

**Comment by remko** [comment]
Updated.

**Comment by hellendag** [comment]
Thanks @remko!

**Comment by hellendag** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**